modules/custom/wim_cvdr/wim_cvdr.admin.inc

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * WIM CVDR file for the admin interface.
+ */
+
+/**
+ * Generate admin form.
+ *
+ * @return array
+ *   Containing the form elements.
+ */
+function _wim_cvdr_admin_page() {
+  $form = array();
+
+  // Create textfield for setting webservice url.
+  $form['wim_cvdr_url'] = array(
+    '#type' => 'textfield',
+    '#maxlength' => 600,
+    '#size' => 200,
+    '#title' => t('CVDR XML URL'),
+    '#description' => t('The full URL to the webservice which outputs an XML file.'),
+    '#required' => TRUE,
+    '#default_value' => variable_get('wim_cvdr_url', ''),
+  );
+
+  // Allow to modify the cache time.
+  $form['wim_cvdr_cache_time'] = array(
+    '#type' => 'select',
+    '#title' => t('Cache time'),
+    '#description' => t('!string1 !string2 !string3', array(
```

Comment:
I don't quite understand the usage for the string replacements here because it means that they are now no longer translatable.

Reply:
just missed :), we got it all from the: [coevorden_cvdr](https://bitbucket.org/goalgorilla/coe/src/master/public_html/sites/all/modules/custom/coevorden_cvdr) and probably we should update/replace this module too after we finished with current one.